src/components/molecules/PaginationTable.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useEffect, ReactElement } from 'react'
```

Comment:
is this new component in use anywhere? Couldn't quickly see it being used

Reply:
It isn't just leaved it there until we sure to use the same component for both. But I'll remove it